### PR TITLE
Ceramic graph updates

### PR DESCRIPTION
**Pull request by Stormgun**
We merged these stylistic changes quite fast with the demo drawing near  but a code review is a good idea

### CODE REVIEW

File: src/js/components/pages/scenegraph.jsx [hunk 1]
```diff
@@ -14,6 +14,10 @@ var SceneListStore = require('../../stores/scene-list-store'); //scene-list-stor
 var SceneStore = require('../../stores/scene-store');
 var DragDropContainer = require('../basic-draggable/drag-drop-container.jsx');
 
+/// TEmp remove
```
Comment (aaronpcz) [inline src/js/components/pages/scenegraph.jsx]: Is this still valid?

**Comment by Stormgun** [comment]
All temporary dev features have been removed, but we should revisit adding an exporter and importer into our utils library

**Comment by Stormgun** [comment]
@aaronpcz  requesting a review

**Comment by Matt95** [comment]
Closed - outdated, version-specific, unknown side effects.

**Pull request closed without merging**